crypto-bigint/src/uint/mul.rs

```diff
@@ -44,7 +44,7 @@ impl<const LIMBS: usize> UInt<LIMBS> {
 
     /// Perform wrapping multiplication, discarding overflow.
     pub const fn wrapping_mul(&self, rhs: &Self) -> Self {
-        self.mul_wide(rhs).0
+        self.mul_wide(rhs).1
```

Comment:
As it were, these tests uncovered a bug

Reply:
We probably should cut a release with this fix and yank the versions with this bug. Even though I've seen this comment, I recently hit this bug and it took me some time to uncover this bug.